code/script.js

```diff
@@ -1,53 +1,153 @@
+    sendSound.play();
+    console.log("Message handled by user:", message);
     chat.innerHTML += `
       <section class="user-msg">
         <div class="bubble user-bubble">
           <p>${message}</p>
         </div>
         <img src="assets/user.png" alt="User" />  
       </section>
-    `
+    `;
     // The else if statement checks if the sender is the bot and if that's the case it inserts
     // an HTML section inside the chat with the posted message from the bot
-  } else if (sender === 'bot') {
+  } else if (sender === "bot") {
+    receiveSound.play();
+    console.log("Message handled by bot:", message);
     chat.innerHTML += `
       <section class="bot-msg">
         <img src="assets/bot.png" alt="Bot" />
         <div class="bubble bot-bubble">
           <p>${message}</p>
         </div>
       </section>
-    `
+    `;
   }
 
   // This little thing makes the chat scroll to the last message when there are too many to
   // be shown in the chat box
-  chat.scrollTop = chat.scrollHeight
+  chat.scrollTop = chat.scrollHeight;
+};
+
+function getInput() {
+  const value = input.value;
+  showMessage(value, "user");
+  input.value = "";
+  return value;
 }
 
 // A function to start the conversation
 const greetUser = () => {
-  // Here we call the function showMessage, that we declared earlier with the argument:
-  // "Hello there, what's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, what's your name?", 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆 and see what happens
-}
+  showMessage("Hello there, what´s your name?", "bot");
+  receiveSound.play();
+};
 
 // Eventlisteners goes here 👇
+const handleNameInput = (event) => {
+  event.preventDefault();
+  // Store the value in a variable so we can access it after we
+  // clear it from the input
+  const name = getInput();
+  values["name"] = name;
+  // showMessage(`Welcome ${name}!`, "bot");
+  setTimeout(showMessage, 1000, `Welcome ${name}!`, "bot");
+  //tar bort så att inte boten frågar igen om namn
+  event.target.removeEventListener("submit", handleNameInput);
+  setTimeout(askAge, 2000);
+};
+//att den frågar om namnet o går vidare till nästa
+function askName() {
+  form.addEventListener("submit", handleNameInput);
+}
+// event.target.removeEventListener("submit", handleNameInput);
+// After 1 second, show the next question by invoking the next function.
+// passing the name into it to have access to the user's name if we want
+// to use it in the next question from the bot.
+// const askNextQuestion = () => {
+// };
+
+const handleAgeInput = (event) => {
+  event.preventDefault();
+  const age = getInput();
+  values["age"] = age;
+  event.target.removeEventListener("submit", handleAgeInput);
+  // Kontrollera om age är en siffra och inte är en tom sträng
+  if (!isNaN(age) && age.trim() !== "") {
+    setTimeout(askGender, 2000);
+    // Visa ett felmeddelande om åldern inte är en siffra
```

Comment:
Nice! Would be cool if you validated the other inputs as well, so that you can't answer "33" on gender for example